backend/apps/predictions/views.py

```
@@ -0,0 +1,255 @@
+
+class TriggerMonitoringAPIView(views.APIView):
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        try:
+            from django_q.tasks import async_task
+            async_task("apps.predictions.tasks.monitor_pr_review_delays")
+            async_task("apps.predictions.tasks.update_reviewer_availability")
+            msg = "Monitoring tasks dispatched asynchronously via Django-Q."
+            details = {}
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)